lib/entities/context/DutchQuoteContext.ts

```diff
@@ -142,6 +142,22 @@ export class DutchQuoteContext implements QuoteContext {
       return null;
     }
 
+    // TODO: remove after reputation system is ready
+    // drop Rfq quote if it's significantly better than classic - high chance MM will fade
+    if (classicQuote && this.rfqQuoteTooGood(quote, classicQuote)) {
+      this.log.info(
+        {
+          tradeType: TradeType[classicQuote.request.info.type],
+          rfqIn: quote.amountIn.toString(),
+          rfqOut: quote.amountOut.toString(),
+          classicIn: classicQuote.amountInGasAdjusted.toString(),
+          classicOut: classicQuote.amountOutGasAdjusted.toString(),
+        },
+        'Rfq quote at least 300% better than classic, skipping'
+      );
+      return null;
+    }
```

Comment:
should we add a metric here so we have better visibility into if this is happening often? maybe an alarm too?